src/release_type.rs

```diff
@@ -0,0 +1,33 @@
+macro_rules! release_type {
```

Comment:
add comment to clarify
```suggestion
// This is solved using the Marco implementation to be able to dynamically add options to the `ReleaseType` enum
// while also generating the corresponding function `ReleaseType::all`, which returns all options.
//
// TODO: check if this can be done less complicated
macro_rules! release_type {
```